source/wireversion-featurelist.rst

```diff
@@ -0,0 +1,54 @@
+====================================
+Server Wire version and Feature List
+====================================
+
+.. list-table::
+   :header-rows: 1
+
+   * - Server version
+     - Wire version
+     - Feature List
+
+   * - 2.4 and before
+     - 0
+     - Text search
```

Comment:
This is a bit misleading. The "text" command was added in 2.4, but doesn't exist in earlier versions. It's probably better not to list any features here.